### PR TITLE
update conference planning

### PR DESCRIPTION
**Pull request by SamVerschueren**
Update the conference planning. Let me know if anything has to be adjusted like this one maybe https://github.com/ng-be/www-ng-be-org/issues/76#issuecomment-262630291.

We can also wait for merging this when we have information of Pascal's talk. Also not sure if the planning is final. So as long as that's not the case, it might not be a good idea of merging this.

### CODE REVIEW

File: src/components/conference-day/conference-day.jade [hunk 1]
```diff
@@ -211,35 +216,82 @@
...
                             .schedule-item__description
-                                p Welcome messages
+                                p The conference opening ceremony
 
                         .schedule-item
                             .schedule-item__time 09:00
                             .schedule-item__description
-                                p Session 1 with various talks
-                                p (Exact talk titles and durations for this session will be listed closer to conference date)
+                                p Keynote
```
Comment (SamVerschueren) [inline src/components/conference-day/conference-day.jade:224]: Something else than just `Keynote`?

File: src/components/conference-day/conference-day.jade [hunk 2]
```diff
@@ -211,35 +216,82 @@
                         .schedule-item
                             .schedule-item__time 08:30
                             .schedule-item__description
-                                p Welcome messages
+                                p The conference opening ceremony
```
Comment (SamVerschueren) [inline src/components/conference-day/conference-day.jade:219]: Or just `Opening ceremony`?

File: src/components/sponsors/sponsors.jade [hunk 3]
```diff
@@ -11,7 +11,7 @@
 
     .sponsor-section.sponsor-section--platinum
 
-        .container
+        .container#ordina
```
Comment (SamVerschueren) [inline src/components/sponsors/sponsors.jade:14]: Added these so we could link to them from the conference planning.

**Review by samvloeberghs — CHANGES_REQUESTED** [review]
Can you make the changes as request in the issue? thanks for your effort @SamVerschueren !!

**Comment by SamVerschueren** [comment]
🎉  Not sure about the styling though. Care to check it out first?

**Comment by jvandemo** [comment]
Thank you @SamVerschueren; much appreciated! 👍 

Meanwhile, Pascal's and Oliver's talk details are available:

Talk title
Angular and React: friends learning from each other

Talk description
React and Angular share a lot of architectural decisions and have inspired each other as well. They are both component oriented, have automated changed detection and can render both on the server as well as on the client side. However, there are also quite a few differences in how they work and which philosophy they follow. 

Based on advanced practical examples we will look at the most interesting differences between those two frameworks. This can help you understand your Angular framework even better and gives you insights into what strengths and what weaknesses it has in certain scenarios.

**Comment by jvandemo** [comment]
Oliver, who will talk together with Pascal, has also sent us his details:

Oliver Zeigermann

Titel: Freelance Web Developer

Bio: Oliver Zeigermann is a developer and architect from Hamburg, Germany. Oliver is the co-author of the best selling German book on React, Redux, React Router, and Universal Rendering. He has also worked in several Angular and React projects using Flow and TypeScript. 

Twitter: @DJCordhose

**Comment by SamVerschueren** [comment]
Ah sweet, thanks @jvandemo. Yesterday @samvloeberghs already added Oliver https://ng-be.org/conference-day#oliver-zeigermann.

Will update the talk description!

**Comment by SamVerschueren** [comment]
Done. Only have to wait until the planning is finalised.

**Comment by jvandemo** [comment]
Awesome, thank you @SamVerschueren! Feel free to ping me if you need anything from me. 👍

**Comment by samvloeberghs** [comment]
Todd motto's description:

`In this talk we’ll take a deeper dive into Angular 2 Reactive forms and utilise the powerful APIs Angular 2 offers for complex forms, validation and much more.`

**Comment by SamVerschueren** [comment]
Made the final changes. Also added Todd's description.

File: src/_build/schedule.less [hunk 4]
```diff
@@ -13,6 +13,12 @@
     padding-left: 1.2rem;
   }
 }
+.schedule-item__talk{
+  &:extend(.schedule-item);
+  .schedule-item__description{
+    background: lighten(@ng-be-yellow, 25%);
+  }
+}
```
Comment (SamVerschueren) [inline src/_build/schedule.less:21]: Maybe this could be done differently?

Comment (samvloeberghs) [reply src/_build/schedule.less:21]: na that's perfect!

**Review by samvloeberghs — APPROVED** [review]

**Comment by SamVerschueren** [comment]
Sweet! Thanks for merging!

**Pull request merged**